src/llmcompressor/args/dataset_arguments.py

```diff
@@ -114,15 +114,15 @@ class DatasetArguments(CustomDatasetArguments):
             "help": "Whether or not to concatenate datapoints to fill max_seq_length"
         },
     )
-    raw_kwargs: Dict = field(
+    raw_kwargs: dict = field(
         default_factory=dict,
         metadata={"help": "Additional keyboard args to pass to datasets load_data"},
     )
-    splits: Union[None, str, List, Dict] = field(
+    splits: None | str | list[str] | dict[str, str] = field(
```

Comment:
I had to update the order in a couple places, otherwise parsing tests were failing. `HfArgumentParser` is super finicky and I wonder if we should migrate from it